build_resolvers/lib/src/resolver.dart

```diff
@@ -249,3 +254,38 @@ Future<void> _createSdkVersionFile(File sdkVersionFile) async {
   await sdkVersionFile.create(recursive: true);
   await sdkVersionFile.writeAsString(Platform.version);
 }
+
+List<int> buildSdkSummary(String dartSdkPath) {
+  // Prepare the Dart SDK.
+  var resourceProvider = PhysicalResourceProvider.INSTANCE;
+  var dartSdkFolder = resourceProvider.getFolder(dartSdkPath);
+  var dartSdk = FolderBasedDartSdk(resourceProvider, dartSdkFolder, true)
+    ..useSummary = false
+    ..analysisOptions = AnalysisOptionsImpl();
+
+  // Try to load the flutter engine _embedder.yaml file and add any missing
```

Comment:
Consider extracting some methods with clear names over commenting blocks of code.